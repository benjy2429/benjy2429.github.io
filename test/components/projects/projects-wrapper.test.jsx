import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ProjectsWrapper from '../../../src/components/projects/projects-wrapper';

const mockProject = {
  title: 'Demo project',
  image: 'project-image.jpg',
  text: 'Project description'
};

describe('Projects wrapper component', () => {
  it('renders a list of projects', () => {
    const component = shallow(<ProjectsWrapper projects={[mockProject]} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
