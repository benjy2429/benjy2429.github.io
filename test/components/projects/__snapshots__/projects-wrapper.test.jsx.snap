// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects wrapper component renders a list of projects 1`] = `
<section
  className="projects"
>
  <div
    className="container"
  >
    <h2
      id="projects"
    >
      Projects
    </h2>
    <p>
      These are some of the projects Iâ€™ve worked on, both at university and in my spare time.
    </p>
    <div
      className="projects-wrapper"
    >
      <Component
        image="project-image.jpg"
        key="Demo project"
        text="Project description"
        title="Demo project"
      />
    </div>
  </div>
</section>
`;
